app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -273,18 +273,71 @@ struct DeviceDelegateOpenXR::State {
 
   void InitializeSwapChainFormats() {
     uint32_t swapchainFormatCount;
-    CHECK_XRCMD(xrEnumerateSwapchainFormats(session, 0, &swapchainFormatCount, nullptr));
-    CHECK_MSG(swapchainFormatCount > 0, "OpenXR unexpected swapchainFormatCount");
+    RETURN_VOID_XR_FAILED(xrEnumerateSwapchainFormats(session, 0, &swapchainFormatCount, nullptr));
```

Comment:
I think the ASSERT is correct, we should not continue if the runtime cannot enumerate swapchain formats

Reply:
I can see crashes if not doing so, so that's why I changed to this. One example is that, in Meta Quest 2, if you start the application in sleep mode (such as start debugging), wait a while and then wake up the device (wear the headset), you can see crashes.

Reply:
Maybe the crash shouldn't be fixed in this way. I will remove this.

Reply:
The crash is fixed in latest commit